dockereve-master/docker-compose.yml

```diff
@@ -13,11 +13,11 @@ nginx:
 
 swagger_ui:
     image: swaggerapi/swagger-ui:v3.0.12
-    environment:
-        - API_URL=http://${HOSTNAME}/docs/api
+    env_file: .env
```

Comment:
I don't think you need to do this explicitly. See: https://docs.docker.com/compose/environment-variables/#the-env-file